src/main/java/mate/academy/internetshop/controllers/CompleteOrderController.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.ShoppingCart;
+import mate.academy.internetshop.service.OrderService;
+import mate.academy.internetshop.service.ShoppingCartService;
+import mate.academy.internetshop.service.UserService;
+
+@WebServlet("/completeOrder")
+public class CompleteOrderController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy.internetshop");
+    private UserService userService = (UserService) INJECTOR.getInstance(UserService.class);
+    private OrderService orderService = (OrderService) INJECTOR.getInstance(OrderService.class);
+    private ShoppingCartService shoppingCartService
+            = (ShoppingCartService) INJECTOR.getInstance(ShoppingCartService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException {
+        ShoppingCart shoppingCart = shoppingCartService.getByUserId(USER_ID);
+        orderService.completeOrder(shoppingCart);
```

Comment:
shopping cart and User)cause you should assign your order to some user after creating